core/new-gui/src/app/workspace/component/result-panel/console-frame/console-frame.component.ts

```diff
@@ -65,10 +65,7 @@ export class ConsoleFrameComponent implements OnInit, OnChanges {
       .getExecutionStateStream()
       .pipe(untilDestroyed(this))
       .subscribe(event => {
-        if (
-          event.previous.state === ExecutionState.BreakpointTriggered &&
-          event.current.state === ExecutionState.Completed
-        ) {
+        if (event.current.state === ExecutionState.Completed) {
```

Comment:
we had this case for previous breakpoint triggered but now it becomes completed, we explicitly handle the case to do nothing. For other states becoming completed, there might be some actions to do. However, now it becomes whenever the state is completed (a.k.a all completed states) we do nothing. I think we may not want this logic right?

Reply:
you're right, removed this if block.